touch-layouts/layouts/the-quarry.json

```diff
@@ -0,0 +1,127 @@
+{
+  "schema_version": 1,
+  "layouts": {
+    "Custom": {
+      "name": "Custom",
+      "content": {
+        "left": {
+          "inner": [
+            {
+              "type": "joystick",
+              "action": "leftTrigger",
+              "actionThreshold": 0.75,

```

Comment:
Have you tried setting this value to `2`? Making it a higher value so it's less likely to be triggered by accident.  

I haven't played this game yet so I don't know if it's better to have a separate sprint button.

Reply:
I changed the value to 2. Plays much better at this setting!